packages/flutter/lib/src/animation/animations.dart

```diff
@@ -49,7 +60,9 @@ class AlwaysStoppedAnimation extends Animation<double> {
   AnimationDirection get direction => AnimationDirection.forward;
 }
 
+/// A mixin to help with building animations that proxy the value of another animation.
 abstract class ProxyAnimatedMixin {
+  /// The animation whose value this animation will proxy.
```

Comment:
We should clearly document in ProxyAnimatedMixin that the `parent` must not change over time. It should be a final field. Otherwise, you'll leak listeners and generally not work.